<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AI with Jetson Nano</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gb_project">
       <property name="title">
        <string>1 - Project generation</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbl_working_space">
            <property name="text">
             <string>Working space</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbl_nb_categories">
            <property name="text">
             <string># of categories</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Select a camera</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="btn_working_space">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="btn_project_ok">
            <property name="text">
             <string>Project OK</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="sb_camera_id">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_nb_categories">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="prefix">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_recording">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>2 - Image recording</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="vl_record"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_split_image">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>3-Image spliting in train, val and test folders</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_upload_to_dl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>4 - upload data to DL machine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_train_model">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>5 - train model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_id_nano">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="prefix">
        <string>jetson nano </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tbn_upload_to_nano">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>6 - upload model to Jetson Nano</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_image">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <property name="text">
        <string>Image from camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
